common/amundsen_common/utils/atlas.py

```diff
@@ -10,12 +10,31 @@ class AtlasStatus:
     DELETED = "DELETED"
 
 
+class AtlasEntityOperation:
+    CREATE = 'CREATE'
+    UPDATE = 'UPDATE'
+
+
 class AtlasCommonParams:
     qualified_name = 'qualifiedName'
     guid = 'guid'
     attributes = 'attributes'
     relationships = 'relationshipAttributes'
     uri = 'entityUri'
+    name = 'name'
```

Comment:
I am not entirely sure we need such fine-grained properties in this class. I'd stick with things that are very atlas specific like `uniqueAttributes` or `createdTimestamp`